src_c/circle.c

```diff
@@ -62,12 +61,12 @@ _pg_circle_set_radius(PyObject *value, pgCircleBase *circle)
 }
 
 static int
-pgCircle_FromObject(PyObject *obj, pgCircleBase *temp)
+pgCircle_FromObject(PyObject *obj, pgCircleBase *out)
 {
     Py_ssize_t length;
 
     if (pgCircle_Check(obj)) {
-        memcpy(temp, &((pgCircleObject *)obj)->circle, sizeof(double) * 4);
+        memcpy(out, &((pgCircleObject *)obj)->circle, sizeof(double) * 4);
```

Comment:
Just do `sizeof(pgCircleBase)`. 4 doubles could be wrong, if the struct's members are aligned to certain widths.